pkg/exporter/grpc_packets.go

```diff
@@ -0,0 +1,79 @@
+package exporter
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/gopacket"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	"github.com/netobserv/netobserv-ebpf-agent/pkg/flow"
+	grpc "github.com/netobserv/netobserv-ebpf-agent/pkg/grpc/packet"
```

Comment:
nit: this is the import order used in the agent
1- standard libs
2- netobserv pkg
3- external packages
this applies here and in all newly created files

Reply:
Sure, https://github.com/netobserv/netobserv-ebpf-agent/pull/291/commits/111c915245d415d29c9cd9710ad79e323207b51e